client/lib/analytics/page-view-tracker/index.jsx

```diff
@@ -67,8 +81,19 @@ export class PageViewTracker extends React.Component {
 	}
 }
 
+const mapStateToProps = state => {
+	const selectedSiteId = getSelectedSiteId( state );
+	const selectedSiteSlug = getSiteSlug( state, selectedSiteId );
+	const currentSlug = getSiteFragment( get( window, 'location.pathname', '' ) );
```

Comment:
We need to check before calling a global var. This will throw otherwise in test envs.
```jsx
const currentSlug = typeof window === 'undefined' ? '' : getSiteFragment( get( window, 'location.pathname', '' ) );
```